developer/src/tike/http/Keyman.Developer.System.ServerAPI.pas

```diff
@@ -111,14 +113,41 @@ class procedure TServerDebugAPI.StopServer;
   LastGetStatusTime := 0;
 end;
 
+class function TServerDebugAPI.LoadFileToString(const Filename: string; var s: string): Boolean;
+var
+  fs: TFileStream;
+  ss: TStringStream;
+begin
+  try
+    fs := TFileStream.Create(TKeymanDeveloperPaths.ServerDataPath + 'pid.json', fmOpenRead);
```

Comment:
We should probably use `Filename` here, or get rid of the `Filename` parameter :smile:

Reply:
Oops!